QandA/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+# Create your models here.
+class user(models.Model):
```

Comment:
As informed we wouldn't need this we might be better just using Django user, since it comes with everything like :- 
- Unique and all integrities.
- Have the ability to give custom permission for some user.
- Have the ability to add superuser.
- Have the ability to mark a user as inactive.
- Have the power of secure passwords, Passwords shouldn't be stored in the plain-text format in DB for obvious reasons.

Reply:
Removed the user table and now making necessary changes in the other models to include the Django User Model